Doc/conf.py

```diff
@@ -296,7 +296,7 @@
 }
 
 # Avoid a warning with Sphinx >= 2.0
```

Comment:
Let's put 4.0 here, because that's when [`root_doc`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-root_doc) was introduced:
```suggestion
# Avoid a warning with Sphinx >= 4.0
```